app/src/main/java/com/codesoom/assignment/TaskNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package com.codesoom.assignment;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
+public class TaskNotFoundException extends RuntimeException{
+  public  TaskNotFoundException( String msg){
+    super(msg);
```

Comment:
띄어쓰기 실수는 흔한 문제이니 신경쓰지 마세요. IDE에 딸려있는 포매터를 사용해보는 방법을 고민해 보세요.
```suggestion
public class TaskNotFoundException extends RuntimeException {
  public TaskNotFoundException(String msg) {
    super(msg);
```